src/index.ts

```diff
@@ -0,0 +1,7 @@
+import { UniversalBot } from "botbuilder"
+
+import { textParserMiddleware } from "./middlewares"
+
+export function loadMarkdownMiddleware(bot: UniversalBot): void {
```

Comment:
I think it would be better to export the middleware. The user would load the middleware in their bot.